src/components/Carousel/styles.module.scss

```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100&display=swap');
```

Comment:
ok, this is a bad practice, we know the following all projects should use only 3 types of fonts, in this case, Montserrat, roboto, and lato.
That sounds good, you need to import this font into the root of the project before checking if it exits, on the base repo, you can use the optimized font that is a new feature of next13 you can read the documentation here -> https://nextjs.org/docs/app/building-your-application/optimizing/fonts  please update the pr using this approach.

Reply:
import from the layout page using NEXTjs property